data_structures/linked-list/.pytest_cache/v/cache/lastfailed

```diff
@@ -1,5 +1,7 @@
 {
   "test_linked_list.py::test_after": true,
+  "test_linked_list.py::test_has_loop": true,
+  "test_linked_list.py::test_has_loop_true": true,
   "test_linked_list.py::test_insert_len": true,
   "test_linked_list.py::test_kthFromEnd": true
 }
```

Comment:
You should update your `.gitignore` to include the `.pytest_cache` directory and files from your OS.